android-libkiwixbuild/app/src/main/cpp/include/searcher.h

```diff
@@ -76,10 +76,10 @@ class Searcher
    * @return true if the reader has been added.
    *         false if the reader cannot be added (no embedded fulltext index present)
    */
-  bool add_reader(Reader* reader);
+  bool add_reader(std::shared_ptr<Reader> reader);
```

Comment:
Those include are the include of libkiwix. They must NEVER be modified. 
They are describing the API of libkiwix. Modifying them will change the **declaration** of how libkiwix is working but it will not change how libkiwix is actually working.

This is why we must not include them if our source code but always import them from the project at build time. (To be sure that what is declared is in sync which how it actually works)

Reply:
Now header will add during compilation will not will be part of this repo.